src/aspire/basis/__init__.py

```diff
@@ -186,7 +186,7 @@ def expand(self, x):
         for isample in range(0, n_data):
             b = self.evaluate_t(x[..., isample])
             # TODO: need check the initial condition x0 can improve the results or not.
-            v[..., isample], info = cg(operator, b, tol=tol)
+            v[..., isample], info = cg(operator, b, tol=tol, atol='legacy')
```

Comment:
Do we know what `atol='legacy'` does? The docs are not very specific here. Will things start breaking if we specify `atol=0`? That setting seems to make sense to me.

Reply:
Legacy is the default we were using. Let me know what you think, can change...

https://github.com/scipy/scipy/blob/master/scipy/sparse/linalg/isolve/iterative.py#L95

Reply:
Legacy behavior looks a bit problematic if `bnrm2` is very small (but not zero). I'm guessing that's why they're changing it. Let's put `atol=0` and see what happens.

Reply:
SGTM. Tests pass with 0. I think it is fair to revisit if we have convergence problem etc.